frontend/javascripts/oxalis/view/action-bar/create_animation_modal.tsx

```diff
@@ -250,6 +250,7 @@ function CreateAnimationModal(props: Props) {
 
     const meshes: RenderAnimationOptions["meshes"] = layerNames.flatMap((layerName) => {
       const meshInfos = state.localSegmentationData[layerName]?.meshes?.[axis] || {};
+      const segments = state.localSegmentationData[layerName]?.segments;
```

Comment:
unfortunately, this will only work for non-volume-tracings. for volume tracings the segments are stored in the volume tracing object. you can use `getSegmentsForLayer` to solve this issue.
also see this [slack discussion](https://scm.slack.com/archives/C5AKLAV0B/p1745920737625269) for the "architecture problem".

Reply:
Good catch. I switched to `getSegmentColorAsRGBA()` is even more suitable and covers both cases.

Reply:
nice :+1: now, we don't need to reimplement the color generation in vx either :+1: